### DISCUSSION TITLE
Version 1.0.1.0-release `<In The Deep Dark Twilight>` edition

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/Halloween#58] [Announcements]
[![Hallowe'en! (HLWN)][SHD:mod]][G:url]  
[![KSP version][K:shd]][K:url] [![License][LIC:shd]][LIC:url] [![Pages][SHD:pgs]][pages]

## Version 1.0.1.0-release `<In The Deep Dark Twilight>` edition

* Released
  * 31 Oct 2023
  * for Kerbal Space Program 1.12.5
  * by [zer0Kerbal](https://github.com/zer0Kerbal)

<div style="border:0.5px solid Tomato; background-color: #bada55; color: #FF0000; text-align:center"><h4>
<b>DO A CLEAN INSTALL:</br> DELETE EXISTING INSTALLATION THEN RE-INSTALL</b></h4><p>Download from <a href="https://www.curseforge.com/kerbal/ksp-mods/Halloween/files">CurseForge</a></p></div>

## Change Summary

* General updating and linting
* Update documentation
* NEW: localization
  * ![Italian (Italiano)](https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/img/IT.png ) Italian (Italiano)
  * partial translation
  * Grazie [Alerika](https://github.com/Alerika)

## See More

* [ChangeLog][chlog] for more details of changes
* [Discussions][discu] or [forums][forum] for discussions and news
* [Known Issues][issue] for known issues and feature requests
* [Marketing Slicks][markt]
* [Pages][pages] for so much more (WIP)
* [Parts Catalog][parts] for part pictures

## Localization

>* ![English](https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/img/EN.png) English
>* ![Norwegian (Norsk)](https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/img/NO.png) Norwegian (Norsk) - takk skal du ha to [tplive](https://github.com/tplive) og [boyum](https://github.com/boyum)
>* ***your translation here***
>
> HELP WANTED: See the [README's in the Localization folder](https://github.com/zer0Kerbal/zer0Kerbal/blob/master/Localization/readme.md) for instructions for adding or improving translations. There is also the [quick start guide(s)](https://github.com/zer0Kerbal/zer0Kerbal/blob/master/Localization/quickstart.md). [GitHub][G:url] push is the best way to contribute. *Additions and corrections welcome!*

<div style="border:0.5px solid Tomato; background-color: #BADA55; color: #FF0000; text-align:center">
  <p><b>red box below is a link to forum post on how to get support</b></p>
  <a href="https://forum.kerbalspaceprogram.com/index.php?/topic/83212-*">
    <p><img src="https://i.postimg.cc/vHP6zmrw/image.png" alt="How to get support"></p></a>
  <p style="color: #000000;">Be Kind: Lithobrake, not jakebrake! Keep your Module Manager up to date</p>
</div>

### How to support this and other great mods by [`zer0Kerbal`][zedK]

Comment, click, like, share, up-vote, subscribe

[![Support][PAYPAL:img]][PAYPAL:url] [![Patreon][PATREON:img]][PATREON:url]

[chlog]: https://raw.githubusercontent.com/zer0Kerbal/Halloween/master/changelog.md "Changelog"
[discu]: https://github.com/zer0Kerbal/Halloween/discussions/ "Discussions"
[forum]: https://forum.kerbalspaceprogram.com/index.php?/topic/205185-*/ "Hallowe'en (HLWN)"
[issue]: https://github.com/zer0Kerbal/Halloween/issues/ "Issue Tracker"
[pages]: https://zer0kerbal.github.io/Halloween/ "GitHub Pages"
[markt]: https://zer0kerbal.github.io/Halloween/Marketing "Marketing Slicks"
[parts]: https://zer0kerbal.github.io/Halloween/PartsCatalog "Parts Catalog"

[SHD:mod]: https://img.shields.io/badge/Hallowe'en%20(HLWN)%20-v1.0.1.0--release-BADA55.svg?style=plastic&labelColor=darkgreen/ "1.0.1.0-release"
[SHD:pgs]: https://img.shields.io/badge/GitHub-Pages-white?style=plastic&labelColor=9cf&logoColor=181717&logo=github/ "GitHub IO"

[G:url]: https://github.com/zer0Kerbal/Halloween/ "GitHub"

[K:url]: http://kerbalspaceprogram.com/ "Kerbal Space Program"
[K:shd]: https://img.shields.io/badge/KSP-1.12.5-blue.svg?style=plastic&labelColor=black/ "Kerbal Space Program"

[LIC:url]: https://creativecommons.org/licenses/by-sa/4.0/ "CC BY-SA 4.0"
[LIC:shd]: https://img.shields.io/badge/License-CC%20BY--SA%204.0-ef9421?labelColor=black&style=plastic&logoColor=ef9421&logo=creativecommons "CC BY-SA 4.0"

[PAYPAL:img]: https://img.shields.io/badge/Buy%20me%20some%20-LFO-BADA55?style=for-the-badge&logo=paypal&labelColor=FFDD00 "PayPal"
[PAYPAL:url]: https://www.paypal.com/donate?hosted_button_id=DC22YHMEJREKL "PayPal"
[PATREON:img]: https://img.shields.io/badge/Patreon%20-Patreonize-FF424D?style=for-the-badge&logo=patreon "Patreon"
[PATREON:url]: https://www.patreon.com/zer0Kerbal/membership "Patreon"

[zedK]: https://forum.kerbalspaceprogram.com/index.php?/profile/190933-*/ "zer0Kerbal"

**Full Changelog**: https://github.com/zer0Kerbal/Halloween/compare/1.0.0.2-release...1.0.1.0-release

<hr /><em>This discussion was created from the release <a href='https://github.com/zer0Kerbal/Halloween/releases/tag/1.0.1.0-release'>Version 1.0.1.0-release `<In The Deep Dark Twilight>` edition</a>.</em>

**Comment by Botnen1** [comment]
Cool!
